### PR TITLE
minor grammatical fixes, can be ignored.

### PR DESCRIPTION
**Pull request by JacobMGEvans**
No major fixes can be ignored if preferred. 

1. Remove changed to removing 
2. Hyphen added JavaScript-powered 

Thank you for reading.

## Checklist:
- [x] My code is tested.
- [x] My code follows the WordPress code style.
- [x] My code has proper inline documentation.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,25 +22,25 @@ Gutenberg looks at the editor as more than a content field, revisiting a layout
...
 
 ## Blocks
 
 Blocks are the unifying evolution of what is now covered, in different ways, by shortcodes, embeds, widgets, post formats, custom post types, theme options, meta-boxes, and other formatting elements. They embrace the breadth of functionality WordPress is capable of, with the clarity of a consistent user experience.
 
-Imagine a custom “employee” block that a client can drag to an About page to automatically display a picture, name, and bio. A whole universe of plugins that all extend WordPress in the same way. Simplified menus and widgets. Users who can instantly understand and use WordPress  -- and 90% of plugins. This will allow you to easily compose beautiful posts like <a href="http://moc.co/sandbox/example-post/">this example</a>.
+Imagine a custom “employee” block that a client can drag to an About page to automatically display a picture, name and bio. A whole universe of plugins that all extend WordPress in the same way. Simplified menus and widgets. Users who can instantly understand and use WordPress  -- and 90% of plugins. This will allow you to easily compose beautiful posts like <a href="http://moc.co/sandbox/example-post/">this example</a>.
```
Comment (ntwb) [inline README.md]: WordPress uses [oxford commas](https://en.wikipedia.org/wiki/Oxford_Comma), we're going to need this comma back, please 😏

**Review by ntwb — CHANGES_REQUESTED** [review]

**Review by ntwb — APPROVED** [review]

**Review by ntwb — CHANGES_REQUESTED** [review]
Thanks @Cloud887, see above

**Comment by Cloud887** [comment]
Glad I could make some minor contribution to this great project. I also prefer the Oxford comma even given a convention choice. 

Thank you.

**Review by ntwb — APPROVED** [review]

**Comment by ntwb** [comment]
Thank you @Cloud887 for your contribution, every little bit helps 🥇

**Comment by ntwb** [comment]
p.s https://github.com/WordPress/gutenberg/commit/0f6dd64e1e5f33d274659eeecf06f150330be8a7

**Pull request merged**